JeopardyScorekeeper/MainForm.Designer.cs

```diff
@@ -63,14 +63,20 @@ private void InitializeComponent()
             this.btnCol6Row3 = new System.Windows.Forms.Button();
             this.btnCol6Row4 = new System.Windows.Forms.Button();
             this.btnCol6Row5 = new System.Windows.Forms.Button();
+            this.textCat1 = new System.Windows.Forms.TextBox();
+            this.textCat2 = new System.Windows.Forms.TextBox();
+            this.textCat3 = new System.Windows.Forms.TextBox();
+            this.textCat4 = new System.Windows.Forms.TextBox();
+            this.textCat5 = new System.Windows.Forms.TextBox();
+            this.textCat6 = new System.Windows.Forms.TextBox();
             this.SuspendLayout();
             // 
             // btnCol1Row1
             // 
             this.btnCol1Row1.BackColor = System.Drawing.Color.Blue;
             this.btnCol1Row1.Font = new System.Drawing.Font("Microsoft Sans Serif", 19.8F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.btnCol1Row1.ForeColor = System.Drawing.Color.White;
-            this.btnCol1Row1.Location = new System.Drawing.Point(13, 13);
+            this.btnCol1Row1.Location = new System.Drawing.Point(12, 107);
```

Comment:
@rwchhs or @RyanLetourneau feeling up to factoring out these coordinates into something like `row1 = 107` and updating the locations so if the row needs to move later on it is a single change to a variable instead of every button separately in a row? It works fine this way but might be nicer later on if the numbers change again.

Reply:
That would be sensible if that was handwritten code, but it's basically all auto-created by Visual Studio - you place the visual components on the form by drag-and-dropping and the IDE generates the code to represent these components (sorry if that sounds condescending, don't know about your background). If you're introducing variables into the *.Designer.cs files, you're messing with VS's ability to give you the drag & drop form editor because it can't handle variables in there.

Additionally, the form editor already gives you the option to move a whole row at once because you can CTRL-click multiple components so that they're selected as one group and then move that whole group together around with your mouse.

Reply:
Ah gotcha. I didn't realize you could do that with VS!